modules/flyte/s3.tf

```diff
@@ -106,3 +106,14 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "flyte_data_encryp
     ]
   }
 }
+
+resource "aws_s3_bucket_cors_configuration" "flyte_data" {
```

Comment:
We've considered the alternative where instead of using CORS policy, instead use an iframe. However, the file renderer for Flows will make a fetch to the S3 bucket pre-signed URL, so the iframe does not seem suitable for the use case.